testing/tests/functions.rs

```diff
@@ -0,0 +1,46 @@
+use askama::Template;
+
+#[derive(Template)]
+#[template(source = "{{ self::simple_test_fn() }}", ext = "txt")]
```

Comment:
I don't think these examples should need the `self::` prefix, since they should be allowed to just use the neighboring scope from the type impls.